.travis.yml

```diff
@@ -16,8 +16,7 @@ deploy:
   provider: npm
   email: hello@trailsjs.io
   api_key:
-    secure: J8ZYI+2vEbILgSalnTi1wZj374kTa+6yY67SW7uMBDKx6AQprN1Qxc00B7OmrI90xTo6Ia2wSzFPjsiPbh8BHnLI8Eoz0SoHAa5iPORx5XztLJN6kQsjeQWNuCBP5wjvDtsYlJLJsNppjun0Ywnl0w6rsFIBfcbHYRHrovntB8qiuGY5ZSq9Rnki0oDgilwIMLCUlOx1ajB+c6oRq59cNn5TKbuGh7Etb1plzsuPLDkrjrtaw9RcDp49P6jJFNCSsKNJgpy8Ki7QRVA52l16SfrRfctAO98NR6+Y+0kjUqbm+CEcAtMnzmF7/Nqi1VLfP9tRYwR0HbNh4X9yrzfOu1ilF8ZHedB0LPaAu0GdTbPJwkXVweueDQJlAYY5X8rcPbXEYUo5aEL082UhSreVFO/Q7FaG+JmbDqnMDzj2cS2zV1ruPfv8C319rSUgKRFe4GZMUcttbqZfAz490W7Y3PAyBeS7gjG8fxGyv/5iq42JYC60CZIllVLxinwRzAff3zSf8xUhCIZFjGiNb9qGaEj8jo9/vNkUfnCGFrshMEmRqUjS6ae/wS5qPlJ4pCfNCMQDNAwbDeVk5Tc9FmPRtd1lA7j63vAqe9zsKnaMzBmYAPY+wEMvOl8n+SxAylKLy+v44H8MX7nykd58qwdOW8kyYWJqAK7QrwxgnvD8n5Y=
+    secure: UJKl70RkuFLJWOJViDkVklykAVYZmiPjlLku2H0VgJqOnnKDr6HW1sOvp7mMouI89WsKY9Mh0j1UHPS5zz5AxsMx22omYExrTlxDhLg7xUMXFh3m3oX7ZiiasKuDtjoXH7d/SwMWThr+c25KmoPnmVUpOcdyKZmY051fAK/AxwOnEZ+XX/v5TCRwrUzuZ+9lOZ3E9FA3Athc7sE9MvVErB+MFXKpgNUSlahhLeYvFhpsNOes38ictVpNY/cF59lzTD3KFx0lC6UamP1b2fgVB/tdm0s8Ws/bxo4MOKb8Q4LiUJbKYuZzU/nsImFC+8MRkcTDS7/qqkd64j0sAq4L5Sei+9z/mIlT7iUBEYma3iU+Zc0Gqn90lwLMloI1oSp6Mf/L+8M8BpQLgT1aNA3cTmUhOg7a9MWEQMajQKAcp975wk0/ztJfVE7PRBrVGmbhZuEN3dpjpQqxSCNyYUNHvydgwPKCUKN06qli8fk80+P7PK+e8op13K/kLIejc2jn/8tXgXsJNNRWCiYu3aMLUy6mNgjFa7Dvm0+xeQ/tMBD85U2eYo61UmJi7aTzEYcR/QsLgFPHE9Ng1h9O/+K/IrbRVa6Q1QY4JJje8CXZzSPBDbVl+PPvAiyxmmnxP09rSvkoAgjbODUzi9XdxuscpCiTq1shr85roNfquBKK1GA=
   on:
     tags: true
     repo: trailsjs/trailpack-express
-    node: stable
```

Comment:
but if you don't specify a node version it will try to publish 3 times ? 4 5 and 6 ?